exe/technologist

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env ruby
+
+require "technologist"
+
+def framework_report(title, frameworks)
+  return if frameworks.empty?
+    puts title
+    frameworks.each do |framework|
+      puts "* #{framework}"
+    end
```

Comment:
A message like "No known frameworks could be detected." for the negative case maybe?